Input box too small on mobile devices

![screen shot 2015-04-10 at 5 29 53 pm](https://cloud.githubusercontent.com/assets/4545925/7087286/4ec3658e-dfa7-11e4-806f-cbfed91e5e77.png)

I think its a good option to make the input box appear at the full length on the screen, and put the JSON and RSS button together in the next line.

Tested the setup which runs clean on
1. Ubuntu 
2. Fedora

Only breaks on Mac OS X with the following error
```sh
npm ERR! notarget No compatible version found: nomnom@'douglasduteil/nomnom#next'
npm ERR! notarget Valid install targets:
```

--- a/html/index.html
+++ b/html/index.html
@@ -69,13 +69,26 @@
       </div>
         <br>
         <br>
-          <form class="col-lg-12">
+          <form class="col-lg-12 hidden-xs">
             <input id="timezoneOffset" type="hidden" name="timezoneOffset" value=""><script>document.getElementById("timezoneOffset").value = new Date().getTimezoneOffset();</script>
             <div class="input-group input-group-lg col-sm-8" style="float:left">
               <input name="q" id="query" type="text" class="center-block form-control input-lg" title="Search Twitter and the loklak network" placeholder="Search Twitter and the loklak network">
               <span class="input-group-btn"><button class="btn btn-lg btn-primary" type="submit" target="_blank" onclick="form.target='_blank'; form.action='/api/search.json?q=' + document.getElementById('query').value;">Get JSON Data</button></span>
             </div>
             <input type="submit" value="Get RSS Feed" class="btn btn-success btn-lg" onclick="form.target='_blank'; form.action='/api/search.rss?q=' + document.getElementById('query').value;"/>
+            <br/><span style="font-size:7pt;">
+              <span class="label label-info">Info</span>
+              Input a search text or "search text" for exact matches; @user finds user mentions, from:user find messages from a user, #hashtag finds a hashtag, near:location finds messages near a location.</span>
+          </form>
+
+          <form class="col-lg-12 hidden-lg hidden-md hidden-sm row">
+            <input id="timezoneOffset" type="hidden" name="timezoneOffset" value="">
+            <script>document.getElementById("timezoneOffset").value = new Date().getTimezoneOffset();</script>
+            <input name="q" id="query" type="text" class="center-block form-control input-lg" title="Search Twitter and the loklak network" placeholder="Search Twitter & the loklak network" style="border-bottom-width: 5px;padding-right: 12px;">
+            <div class="input-group input-group-lg row col-sm-12" style="float:left">
+              <span class="input-group-btn"><button class="btn btn-lg btn-primary" type="submit" target="_blank" onclick="form.target='_blank'; form.action='/api/search.json?q=' + document.getElementById('query').value;">Get JSON Data</button>
+              <input type="submit" value="Get RSS Feed" class="btn btn-success btn-lg" onclick="form.target='_blank'; form.action='/api/search.rss?q=' + document.getElementById('query').value;"/></span>
+            </div>
             <br/><span style="font-size:7pt;">
               <span class="label label-info">Info</span>
               Input a search text or "search text" for exact matches; @user finds user mentions, from:user find messages from a user, #hashtag finds a hashtag, near:location finds messages near a location.</span>

